WordPress/src/main/res/layout/comment_edit_activity.xml

```diff
@@ -1,48 +1,68 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:fillViewport="true"
-    android:clipToPadding="false"
-    android:paddingBottom="@dimen/margin_large"
-    android:paddingTop="@dimen/margin_large"
-    android:background="@android:color/white"
-    android:scrollbarStyle="outsideInset"
-    android:paddingStart="@dimen/margin_extra_large"
-    android:paddingEnd="@dimen/margin_extra_large">
-
-    <RelativeLayout
+    android:orientation="vertical">
+
+    <include layout="@layout/toolbar_main" />
+
+    <ScrollView
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="vertical">
+        android:layout_height="match_parent"
+        android:clipToPadding="false"
+        android:fillViewport="true"
+        android:paddingStart="@dimen/margin_extra_large"
+        android:paddingTop="@dimen/margin_medium"
+        android:paddingEnd="@dimen/margin_extra_large"
+        android:paddingBottom="@dimen/margin_large"
+        android:scrollbarStyle="outsideInset">
 
-        <LinearLayout
-            android:id="@+id/edit_comment_container"
+        <RelativeLayout
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:orientation="vertical">
 
-            <EditText
-                android:id="@+id/edit_comment_content"
+            <LinearLayout
+                android:id="@+id/edit_comment_container"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:gravity="top"
-                android:hint="@string/hint_comment_content"
```

Comment:
I'm pretty sure it is wanted but I prefer to cross check with you. I saw from one of your comment that we wanted to remove the horizontal line, but it seems we are also losing the hint, like in the following image:

![image](https://user-images.githubusercontent.com/47797566/72810425-d0260880-3c5d-11ea-949e-48b6a229016c.png)

Reply:
Hints in `TextInputLayout` look a bit different - there is a label that goes to the top when you start typing and it doesn't really match the look. Considering that you will always open edit comment screen with some sort of comment (and there is an error if a comment is empty) I decided that we can get rid of it.

Reply:
Thanks for the explanation Klym 😊!